packages/utils/sets-utils/src/cell-set-utils.js

```diff
@@ -509,30 +509,44 @@ export function treeToCellPolygonsBySetNames(
  * Given a tree with state, get the sizes of the
  * sets currently marked as "visible".
  * @param {object} currTree A tree object.
+ * @param {array} allNamePaths Array of all paths.
  * @param {array} selectedNamePaths Array of arrays of strings,
- * representing set "paths".
+ * representing selected paths.
  * @param {object[]} setColor Array of objects with the
  * properties `path` and `color`.
  * @param {string} theme "light" or "dark" for the vitessce theme
  * @returns {object[]} Array of objects
  * with the properties `name`, `size`, `key`,
  * and `color`.
  */
-export function treeToSetSizesBySetNames(currTree, selectedNamePaths, setColor, theme) {
+export function treeToSetSizesBySetNames(
```

Comment:
Can you please add a few unit tests for this function (in particular for some of the edge cases)? I am not sure whether we had one already, but now that the functionality is slightly more complex it would be great to have.

Reply:
yes, very good point! I will add some

Reply:
I have added 2 tests that cover most of the cases.